scripts/update_csv_from_firebase.py

```diff
@@ -135,7 +139,7 @@ def get_images_already_being_labeled():
 
         # weird extra folder case
         if d_image["input_img"].split("/")[3].startswith("2021"):
-            country.append(None)
+            country.append(" ")
```

Comment:
This `None` may be important, I would verify and potentially leave it as is while changing the type hint to `country: List[Optional[str]]`